Holovibes/includes/gui/windows/HoloWindow.hh

```diff
@@ -4,8 +4,8 @@
  */
 #pragma once
 
-#include "icompute.hh"
 #include "RawWindow.hh"
+#include "display_queue.hh"
```

Comment:
Inverted include order

Reply:
Well in terms of alphabetical order, maybe there are dependencies I am not seeing